cloudify_kubernetes/decorators.py

```diff
@@ -32,7 +31,6 @@
 from .k8s import (CloudifyKubernetesClient,
                   KubernetesApiAuthenticationVariants,
                   KubernetesApiConfigurationVariants,
-                  KuberentesApiInitializationFailedError,
```

Comment:
Is this not possible?

Reply:
Is is, but it doesn't require any special handling rather than the one we do for `RecoverableException`